cli/config/config.go

```diff
@@ -95,6 +95,11 @@ func GetSeedImages() []string {
 	}
 }
 
+// SetSeedImages sets the list of image string specified in the package.
```

Comment:
Confused on what this is doing here?   Composability shouldn't really touch seed images as that's a feature exclusive to init packages and in the latest multi-distro PR will only be a string instead of slice.  Can you clarify why this was needed?

Reply:
To further clarify, there will never be more than a single initial seed image in the future and that should be outside the scope of the composability feature.

Reply:
Ok great! I wanted validation on that, I was thinking it may not be needed, but since they were pulled in package create I went ahead and added but was going to ask you about that.

Reply:
@jeff-mccoy Removed all references to the seed images in the compose methods in commit [08943fa](https://github.com/defenseunicorns/zarf/pull/351/commits/08943fa1f8b182a27bb98342315cd78f24b4892c)